integration/common/openlineage/common/utils.py

```diff
@@ -50,3 +50,17 @@ def get_from_multiple_chains(source: Dict[str, Any], chains: List[List[str]]) ->
         if result:
             return result
     return None
+
+
+def parse_single_arg(args, keys: List[str]) -> Optional[str]:
```

Comment:
Why not use [`argparse`](https://docs.python.org/3/library/argparse.html)?

Reply:
I thought about it, but it's far too complex for this simple use case and requires disabling a lot of options, like help, exit on failure or usage. This function does exactly what we need, which is transparent searching for keys in passed list.